sydney/sydney.py

```diff
@@ -91,10 +95,31 @@ def _build_ask_arguments(self, prompt: str) -> dict:
             "responsible_ai_policy_235",
             "enablemm",
             "dv3sugg",
+            "iyxapbing",
+            "iycapbing",
+            "galileo",
+            "saharagenconv5",
+            "h3imagntvwcp",
+            "seqnumtts",
+            "log2sph",
+            "savememfilter",
+            "uprofgen",
+            "uprofupd",
+            "uprofupdasy",
+            "eredirecturl",
         ]
         for style in style_options:
             options_sets.append(style.strip())
-
+        blob_data = {
+            "blobId": None,
+            "processedBlobId": None
+        }
+        if attachment:
+            attachment_info = await self._uploadAttachment(attachment)
```

Comment:
I think it might be best to keep this outside of the _build_ask_arguments method so it stays as a sync method instead of async

So, get the attachment info outside of it and then pass it as an argument